composer.json

```diff
@@ -13,19 +13,19 @@
     "require": {
         "php": "^7.1",
         "doctrine/dbal": "^2.6",
-        "symfony/console": "~3.3|^4.0",
-        "ocramius/proxy-manager": "^1.0|^2.0"
+        "ocramius/proxy-manager": "^1.0|^2.0",
+        "symfony/console": "~3.3|^4.0"
```

Comment:
Does it still make sense to support these old versions when they are just ignored because of the PHP version requirement (might be unrelated to this PR, though)?

Reply:
yes, thought so too, didn't want to do here :smile: